src/main/webapp/map.js

```diff
@@ -9,16 +8,18 @@ var coffeeShopInfo = [];
 /** Creates a map that shows all coffee shops around the user. */
 function createMap() {
   // Set default location at new york city
-  var newyork = new google.maps.LatLng(40.7128, -74.0060);
+  var defaultLcation = new google.maps.LatLng(40.7128, -74.0060);
```

Comment:
defaultLocation is more readable than defaultLcation - IMO the 1 char saving is not worthwhile :)